src/tools/pmempool/common.c

```diff
@@ -266,7 +266,8 @@ util_parse_range_to(char *str, struct range *rangep, struct range *entirep)
 static int
 util_parse_range_number(char *str, struct range *rangep, struct range *entirep)
 {
-	util_parse_size(str, &rangep->first);
+	if (util_parse_size(str, &rangep->first))
```

Comment:
In other places, it looks like: `if (util_parse_size(str, &rangep->first) != 0)`
And assuming the error is indicated by "-1", then maybe it would be better to use "< 0"?